src/components/alphabet/models/alphabet.js

```diff
@@ -0,0 +1,368 @@
+import AlphabetItem from './alphabet-item';
+
+export default function getAlphabet() {
+    return [
+        new AlphabetItem(
```

Comment:
This urls, letter, names should in some kind of config to strip it out of normal code.